scipy/stats/_stats_py.py

```diff
@@ -4477,6 +4477,10 @@ def pearsonr(x, y, *, alternative='two-sided'):
                           alternative=alternative)
 
 
+FisherExactResult = _make_tuple_bunch('FisherExactResult',
+                                      ['statistic', 'pvalue'], [])
```

Comment:
I would prefer to have a common "tuple bunch" as I said in the other PR from @mdhaber. Otherwise I am +1 on doing this.

Reply:
Ok, what should the name be? `StatsTestResult`?

Reply:
Might be too general. What about `HypothesisTestResult`? Not sure if there would be something that would just say p-value and statistic. `SignificanceTestResult` maybe?

Reply:
Is `combine_pvalues` well-described by "hypothesis test"? Not sure, but I like a slight tweak of what you wrote: I think `SignificanceResult` is a good name for an object with just attributes `statistic` and `pvalue`.